app/cli/internal/action/util.go

```diff
@@ -0,0 +1,34 @@
+package action
+
+import (
+	"errors"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// LoadFileOrURL loads a file from a local path or a URL
+func LoadFileOrURL(fileRef string) ([]byte, error) {
```

Comment:
do you need to expose this function?

Reply:
Thank you, good catch. Is is in the same `action` package.